README.md

```diff
@@ -71,7 +71,8 @@ Most basic usage just setting required arguments:
 
 ```hcl
 module "terraform-aws-vpc" {
-  source  = "git@github.com:mineiros-io/terraform-aws-vpc.git?ref=v0.6.0"
+  source  = "mineiros-io/vpc/aws"
+  version = "~> 0.7.0"
```

Comment:
Why this way as opposed to the other way?

Reply:
This is the recommended way once a Terraform module has been published to https://registry.terraform.io/
We're using Git as protocol only for non-published modules or private modules

Reply:
Ah, this makes sense thank you